zproject/urls.py

```diff
@@ -78,7 +79,7 @@
         zerver.views.zephyr.webathena_kerberos_login,
         name='zerver.views.zephyr.webathena_kerberos_login'),
 
-    url(r'^accounts/password/reset/$', password_reset,
+    url(r'^accounts/password/reset/$', rate_limit()(password_reset),
```

Comment:
@timabbott Is this actually needed?

My reading of https://docs.djangoproject.com/en/1.11/topics/auth/default/#django.contrib.auth.views.password_reset
is that this will send the actual user an email, which they can ignore -- and it's intended to leak no information to the requester in any case. It isn't a password-check path.